app/components/TextWithIcon/TextWithIcon.css

```diff
@@ -6,3 +6,7 @@
 .textContainer {
   width: max-content;
 }
+
+.overflowWrap {
+  overflow-wrap: anywhere;
+}
```

Comment:
This feels dangerous. Have you checked how this affects other uses of the component? Like on the event detail page

Reply:
Yeah, I quickly checked through all uses now and couldn't spot any problems